logger/raven_logger.py

```diff
@@ -104,7 +104,11 @@ def capture(self, shell_args):
             extra['environment'] = dict([item.split('=', maxsplit=1) for item in shell_args.env.split('\n')])
 
         if shell_args.stderr:
-            extra['stderr'] = shell_args.stderr
+            # raven only accepts some number of list items
+            #  but you probably want the end of the message
+            list_max_length = getattr(self._client, 'list_max_length', 50)
```

Comment:
@NachoSoto Made it more defensive since we don't "control" the raven version and didn't type gertattr.